BUILD

```diff
@@ -436,7 +436,7 @@ grpc_cc_library(
         "src/core/lib/channel/handshaker_registry.c",
         "src/core/lib/channel/http_client_filter.c",
         "src/core/lib/channel/http_server_filter.c",
-        "src/core/lib/channel/max_age_filter.c",
+        "src/core/ext/filters/max_age/max_age_filter.c",
```

Comment:
Do we want `grpc_base` depending on code in `ext`?

It looks like in `build.yaml`, you moved `max_age_filter` into its own target.